internal/validations/clusterlogforwarder/validate_filters.go

```diff
@@ -0,0 +1,51 @@
+package clusterlogforwarder
+
+import (
+	"regexp"
+
+	loggingv1 "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/validations/clusterlogforwarder/conditions"
+	"github.com/openshift/cluster-logging-operator/internal/validations/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// TODO: THIS NEEDS TO CHANGE!
+var pathExpRegex = regexp.MustCompile("^.[a-zA-Z_-][a-zA-Z0-9_-]*(.[a-zA-Z_-][a-zA-Z0-9_-]*|/[a-zA-Z_-][a-zA-Z0-9_-]*)*$")
```

Comment:
only support dot delimited path if this does more

Reply:
Changed to do that. I've also included paths such as `.kubernetes.labels."foo-bar/baz"` because any segment that is not alphanumeric or `_` must be quoted.